### DISCUSSION TITLE
can't get selected option on select2 ajax inside repeatable element

### DISCUSSION THREAD

**Discussion by rickyviz** [Laravel-Backpack/community-forum#256] [Q&A (Help)]
My controller
![image](https://user-images.githubusercontent.com/89818695/169093362-d1b1caaf-49b9-44dd-96fc-2aa7f1ee67d1.png)
My Model Supplier Inovice
![image](https://user-images.githubusercontent.com/89818695/169092920-0407f887-fb54-4170-b02f-66161fbf9d49.png)
Model for supplier category pirce (1-N)
![image](https://user-images.githubusercontent.com/89818695/169058859-f5f97e37-bdb5-4cfa-ab9a-f95d734eae73.png)
i got value amount, but not category_id (ON EDIT)
![image](https://user-images.githubusercontent.com/89818695/169058937-2acaf02b-ea65-4f17-b7df-1c5a0561ed03.png)
All option showing on ajax call
![image](https://user-images.githubusercontent.com/89818695/169066259-2c36d0cd-9562-48ce-ab36-5ca841360d7e.png)
i make new field inside repeatable , with filed type select, and it showing properly on edit, so this not working for select2_ajax
![image](https://user-images.githubusercontent.com/89818695/169093190-cbc4dc24-567a-480b-abc8-ea50bcab12d2.png)
![image](https://user-images.githubusercontent.com/89818695/169093240-c7707faa-f6be-4d1e-815d-dd066e94ef34.png)

**Comment by jcastroa87** [comment]
Hello @rickyviz did you try this in our updated version? select2 is working normally.

Cheers.
